pyEpiabm/pyEpiabm/core/population.py

```diff
@@ -50,3 +54,24 @@ def total_people(self):
         for cell in self.cells:
             count += len(cell.persons)
         return count
+
+    def enqueue_vaccine(self, priority, counter, person: Person):
+        """Add person to queue for processing when mass vaccination
+        begins.
+
+        Parameters
+        ----------
+        priority : int
+            Priority level of 1, 2, 3, or 4 to prioritise by age and
+            whether carehome resident. 1 being highest priority and
+            4 being lowest.
+        counter : int
+            Counter to prioritise by order of addition within each
+            priority group. Generated successively onn addition of
```

Comment:
```suggestion
            priority group. Generated successively on addition of
```

Reply:
Yes they are ordered by addition within each group, but they won't be ordered by cell because after reading everyone in from each cell the entire list is shuffled.

Reply:
Okay that's good then!